src/routes/_error.svelte

```diff
@@ -1,18 +1,48 @@
 <script>
-  export let status;
-  export let error;
-
-  const dev = process.env.NODE_ENV === 'development';
+  import Layout from '@/layouts/default.svelte';
+  import Button from 'ui/button.svelte';
 </script>
 
 <svelte:head>
-  <title>{status}</title>
-</svelte:head>
+  <title>Page not found – Innopoints</title>
 
-<h1>{status}</h1>
+  <link rel="stylesheet" href="/css/page-components/header.css">
+  <link rel="stylesheet" href="/css/page-components/404.css">
+  <link rel="stylesheet" href="/css/page-components/footer.css">
 
-<p>{error.message}</p>
+  <!-- Home styles -->
+  <link rel="preload" href="/css/home/main.css" as="style" />
+  <link rel="preload" href="/css/home/header.css" as="style" />
+  <link rel="preload" href="/css/home/tagline.css" as="style" />
+  <link rel="preload" href="/css/home/how-to.css" as="style" />
+  <link rel="preload" href="/css/home/options.css" as="style" />
+  <link rel="preload" href="/css/home/store.css" as="style" />
+  <link rel="preload" href="/css/home/contacts.css" as="style" />
+</svelte:head>
 
-{#if dev && error.stack}
-  <pre>{error.stack}</pre>
-{/if}
+<Layout>
+  <div class="material container-404 padded">
+    <div class="text">
+      <h1>Nothing here</h1>
+      <p class="subtitle">That rabbit must have given you the wrong lead.</p>
+      <Button href="/" isFilled>
+        <svg src="/images/icons/home.svg" class="icon mr" />
+        take me home
+      </Button>
+    </div>
+    <img src="/images/404/web-traveller.svg" width="240" height="387" class="picture" alt="">
+    <h2>Feeling lucky?</h2>
```

Comment:
This heading is not visible on desktop.
The links' images do not represent the destination, and there is no additional label/description

Reply:
Markup problems 🤷‍♂️